test/navigation/routeDetails/LoadingNavigatorSpec.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation.routeDetails
+
+import base.SpecBase
+import generators.{Generators, RouteDetailsUserAnswersGenerator}
+import models._
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{verify, when}
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import services.CountriesService
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class LoadingNavigatorSpec extends SpecBase with ScalaCheckPropertyChecks with Generators with RouteDetailsUserAnswersGenerator {
+
+  private val navigator = new LoadingNavigator(ctcCountryCodes, customsSecurityAgreementAreaCountryCodes)
+
+  "Location Of Goods Navigator" - {
+
+    "when in NormalMode" - {
+
+      val mode = NormalMode
+
+      "when answers complete" - {
+        "must redirect to loading check your answers" ignore {
+          forAll(arbitraryLocationOfGoodsAnswers(emptyUserAnswers)) {
```

Comment:
Not a huge issue because the test is being ignored but it could be worth creating a `arbitraryLoadingAnswers` and adding the pages as a separate def in `UserAnswersEntryGenerators`